tests/test_dependencies.py

```diff
@@ -29,11 +30,7 @@ def test_pinned_twisted_version(self):
         if not os.environ.get('_SCRAPY_PINNED', None):
             self.skipTest('Not in a pinned environment')
 
-        tox_config_file_path = os.path.join(
-            os.path.dirname(__file__),
-            '..',
-            'tox.ini',
-        )
+        tox_config_file_path = Path(__file__).parent / '..' / 'tox.ini'
```

Comment:
```suggestion
        tox_config_file_path = Path(__file__).parent.parent / 'tox.ini'
```

Reply:
The old code was like this so I didn't change it. `'..'` and `.parent` are slightly different if there are symbolic links. It'd probably be okay to change it here.